streamlit-hello-world-app/app.py

```diff
@@ -0,0 +1,36 @@
+import os
+from databricks import sql
+from databricks.sdk.core import Config
+import streamlit as st
+import pandas as pd
+import numpy as np
+
+st.set_page_config(layout="wide")
+
+# def sqlQuery(query: str) -> pd.DataFrame:
+#     # ensure the right environment variables are set
+#     def defined(var: str) -> bool: return os.getenv(var) is not None
+#     assert defined('DATABRICKS_WAREHOUSE_ID') and os.getenv('DATABRICKS_WAREHOUSE_ID') != "<your warehouse ID>", "To use SQL, set DATABRICKS_WAREHOUSE_ID in app.yaml. You can find your SQL Warehouse ID by navigating to SQL Warehouses, clicking on your warehouse, and then looking for the ID next to the Name."
```

Comment:
Should we remove these comments?

Reply:
I've talked to Nick and he wants to keep the comments in, because he feels like these are the first things people will do after getting a running app.